tests/spread/tutorials/basic/task.yaml

```diff
@@ -0,0 +1,31 @@
+summary: smoke test for Chisel
+
+execute: |
+  set -eux
+
+  rootfs_folder=rootfs
+  mkdir -p $rootfs_folder
+  chisel cut --release ubuntu-`lsb_release -a | grep Release | awk -F' ' '{print $NF}'` \
```

Comment:
The host release and the chisel release should be independent I think. We may get help from Spread "variants" to make this easier.

Reply:
Using variants now